Arrays/957PrisonCellsAfterNDays.py

```
@@ -0,0 +1,23 @@
+          return new_cells
+      def prisonAfterNDays(self, cells: list[int], n: int) -> list[int]:
+          hash_map={}
+          for i in range(n):
+            #   detect the loop
+               temp=str(cells)
+               if temp in hash_map:
+                  loop_len=i-hash_map[temp]
+                  return self.prisonAfterNDays(cells,(n-i)%loop_len)
+               else:
+                   hash_map[temp]=i 
+                   cells=self.nextCells(cells)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)